getting-started/android-frontend-with-appengine-backend/Backend/src/main/kotlin/com/google/cloud/kotlin/emojify/EmojifyController.kt

```diff
@@ -87,13 +88,17 @@ class EmojifyController(@Value("\${storage.bucket.name}") val bucketName: String
     }
 
     val emojiBufferedImage = mapOf(
-        Emoji.JOY to stream("emojis/joy.png"),
-        Emoji.ANGER to stream("emojis/anger.png"),
-        Emoji.SURPRISE to stream("emojis/surprise.png"),
-        Emoji.SORROW to stream("emojis/sorrow.png"),
-        Emoji.NONE to stream("emojis/none.png")
+        Emoji.JOY to retrieveEmoji("joy.png"),
+        Emoji.ANGER to retrieveEmoji("anger.png"),
+        Emoji.SURPRISE to retrieveEmoji("surprise.png"),
+        Emoji.SORROW to retrieveEmoji("sorrow.png"),
+        Emoji.NONE to retrieveEmoji("none.png")
     )
 
+    private final fun retrieveEmoji(name: String): BufferedImage {
```

Comment:
Looks like we're doing this so we don't have to read from GCS every time? Really good idea :D

Although it wouldn't be a bad idea to rename `stream` below to `streamFromCloudStorage`